Cannot run arquillian testsuite with quarkus-embedded due to dependency conflict

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

testsuite

### Describe the bug

[ERROR] The wrapped NoClassDefFoundError is likely caused by the versions of JUnit jars on the classpath/module path not being properly aligned. 
Please ensure consistent versions are used (see https://docs.junit.org/5.13.4/user-guide/#dependency-metadata).
The following conflicting versions were detected:
- org.junit.jupiter.api: 5.13.4
- org.junit.jupiter.engine: 5.13.4
- org.junit.platform.commons: 1.13.4
- org.junit.platform.engine: 1.13.4
- org.junit.platform.launcher: 1.13.4
- org.junit.vintage.engine: 5.9.3

### Version

main

### Regression

- [ ] The issue is a regression

### Expected behavior

tests are running

### Actual behavior

tests are not running

### How to Reproduce?

mvn -f testsuite/integration-arquillian/tests/base/pom.xml -Pauth-server-quarkus-embedded clean install -Dtest=OrganizationAuthenticationTest

### Anything else?

_No response_

--- a/testsuite/integration-arquillian/tests/base/pom.xml
+++ b/testsuite/integration-arquillian/tests/base/pom.xml
@@ -733,7 +733,6 @@
                 <surefire.memory.Xmx>2048m</surefire.memory.Xmx>
                 <surefire.memory.metaspace>512m</surefire.memory.metaspace>
                 <surefire.memory.metaspace.max>1024m</surefire.memory.metaspace.max>
-                <junit-vintage-engine.version>5.9.3</junit-vintage-engine.version>
             </properties>
             <dependencies>
                 <dependency>
@@ -751,7 +750,6 @@
                 <dependency>
                     <groupId>org.junit.vintage</groupId>
                     <artifactId>junit-vintage-engine</artifactId>
-                    <version>${junit-vintage-engine.version}</version>
                     <scope>test</scope>
                 </dependency>
                 <dependency>

